component/src/main/java/org/wso2/carbon/event/input/adapter/mqtt/MQTTEventAdapterFactory.java

```diff
@@ -0,0 +1,156 @@
+        password.setDisplayName(resourceBundle.getString(
+                org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants
+                        .ADAPTER_CONF_PASSWORD));
+        password.setSecured(true);
+        password.setEncrypted(true);
+        password.setRequired(false);
+        password.setHint(resourceBundle.getString(
+                org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants.ADAPTER_CONF_PASSWORD_HINT));
+        propertyList.add(password);
+
+        //Broker SSL Connection
+        Property connectionSSLEnabled = new Property(
+                org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants.ADAPTER_CONNECTION_SSL_ENABLED);
+        connectionSSLEnabled.setDisplayName(resourceBundle.getString(
+                org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants
+                                                                             .ADAPTER_CONNECTION_SSL_ENABLED));
+        connectionSSLEnabled.setRequired(false);
+        connectionSSLEnabled.setOptions(new String[]{"true", "false"});
+        connectionSSLEnabled.setDefaultValue("false");
+        connectionSSLEnabled.setHint(resourceBundle.getString(
+                org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants
+                                                                      .ADAPTER_CONNECTION_SSL_ENABLED_HINT));
+        propertyList.add(connectionSSLEnabled);
+
+        //Broker clear session
+        Property clearSession = new Property(
+                org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants.ADAPTER_CONF_CLEAN_SESSION);
+        clearSession.setDisplayName(
+                resourceBundle.getString(
+                        org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants.ADAPTER_CONF_CLEAN_SESSION));
+        clearSession.setRequired(false);
+        clearSession.setOptions(new String[]{"true", "false"});
+        clearSession.setDefaultValue("true");
+        clearSession.setHint(resourceBundle.getString(
+                org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants.ADAPTER_CONF_CLEAN_SESSION_HINT));
+        propertyList.add(clearSession);
+
+        // set clientId
+        Property clientId = new Property(
+                org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants.ADAPTER_CONF_CLIENTID);
+        clientId.setDisplayName(resourceBundle.getString(
+                org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants.ADAPTER_CONF_CLIENTID));
+        clientId.setRequired(false);
+        clientId.setHint(resourceBundle.getString(
+                org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants.ADAPTER_CONF_CLIENTID_HINT));
```

Comment:
Check the code formatting and use wso2 standard formatting. Check this throughout the code.